oauth2_http/pom.xml

```diff
@@ -65,6 +65,16 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
```

Comment:
GAX has converted to use ThreeTenBP instead of Joda - see this PR: https://github.com/googleapis/gax-java/pull/294